### PR TITLE
3.1.0 Release

### PR DESCRIPTION
**Pull request by webron**
Changelog
---------

See [3.1.0-rc1](https://github.com/OAI/OpenAPI-Specification/releases/tag/3.1.0-rc1) for previous changes in 3.1.0, including the explanation of why there are breaking changes.

### Additions

- Added the `jsonSchemaDialect` top-level field to allow the definition of a default $schema value for Schema Objects.

### Updates

- Updated some links to more accurate locations.
- Updates JSON Schema support to the latest [2020-12 draft](https://tools.ietf.org/html/draft-bhutton-json-schema-00).
- Revamped relative reference resolution under both URIs and URLs.
- Reworked file upload description to take into account new JSON Schema capabilities. This contains breaking changes.
- Both `x-oai-` and `x-oas-` prefixes for Specification Extensions are now reserved to be defined by the OpenAPI Initiative.

### Clarifications

- Path parameter values cannot contain the unescaped characters `/`, `?` or `#`.
- Further explanation of where Reference Object and JSON Schema's reference should be used.
- Unified wording when values are URLs/URIs.
- Reworded Path Item's `$ref` to take into account reference and component changes.
- Fixed some examples.
- Minor text changes to improve consistency and readability.
- The description of the Reference Object has been updated to further clarify its behavior.
- Further updated Schema Object's description to take into account the latest draft, and the default use of https://spec.openapis.org/oas/3.1/dialect/base as the default OAS dialect.
- Reworded "Schema Vocabularies" to "Schema dialects"

### CODE REVIEW

**Comment by webron** [comment]
Requires https://github.com/OAI/OpenAPI-Specification/pull/2461 to be merged prior to this.

**Comment by webron** [comment]
@OAI/tsc - This PR requires an official vote before release. Please vote with 👍 or 👎.

File: versions/3.1.0.md [hunk 1]
```diff
@@ -133,23 +136,18 @@ In order to preserve the ability to round-trip between YAML and JSON formats, YA
 
 ### <a name="documentStructure"></a>Document Structure
 
-An OpenAPI document MAY be made up of a single document or be divided into multiple, connected parts at the discretion of the user. In the latter case, `$ref` fields MUST be used in the specification to reference those parts as follows from the [JSON Schema](https://json-schema.org) definitions.
+An OpenAPI document MAY be made up of a single document or be divided into multiple, connected parts at the discretion of the user. In the latter case, [`Reference Objects`](#referenceObject) and [`Schema Object`](#schemaObject) `$ref` keywords are used.
```
Comment (earth2marsh) [inline versions/3.1.0.md:139]: nit: `author` might be more apt than `user`, as generally speaking they're the category of user who determines the single or multipart choice.

**Review by MikeRalphson — APPROVED** [review]
LGTM. :shipit:

**Review by Cu2y — COMMENTED** [review]
no changes

**Review by issamhaimour — COMMENTED** [review]
Issam 2

**Comment by WNorth97** [comment]
wwon

**Pull request merged**